internal/pkg/agent/application/upgrade/step_mark_test.go

```diff
@@ -25,6 +25,8 @@ import (
 	agtversion "github.com/elastic/elastic-agent/pkg/version"
 )
 
+var noopLocker = &dummyLocker{}
```

Comment:
maybe I am missing something but this doesn't have to be a `dummyLocker` but it can be an actual `noopLocker`? 🙂

Reply:
yes race condition, dummy was first